R/step_adjust_latency.R

```diff
@@ -267,9 +299,9 @@ print.step_adjust_latency <-
     } else {
       terms <- x$terms
     }
-    if (!is.null(x$as_of)) {
+    if (!is.null(x$forecast_date)) {
```

Comment:
Printing looks very strange. This is the example recipe above:
<img width="778" alt="Screenshot 2024-06-14 at 10 08 19" src="https://github.com/cmu-delphi/epipredict/assets/20528078/423161e2-17e4-45d2-8361-964760979e0c">

Reply:
oh I didn't have any tests for this and changed the format. Surprised printing wasn't throwing errors. I'll let you know when I think this is fixed, leaving open